deploy/src/main/deploy/helm/templates/hono-adapter-amqp/hono-adapter-amqp-vertx-deployment.yaml

```diff
@@ -14,7 +14,7 @@ spec:
     spec:
       containers:
       {{- include "hono.jaeger.agent" . | indent 6 }}
-      - image: {{ .Values.honoContainerRegistry }}/eclipse/hono-adapter-amqp-vertx:{{ .Values.honoImageTag }}
+      - image: {{ .Values.honoContainerRegistry }}/{{ .Values.honoImageOrgName }}/hono-adapter-amqp-vertx:{{ .Values.honoImageTag }}
```

Comment:
instead of using multiple variables to assemble the image name from, I had already thought about using a single property for the complete image name, e.g. `adapters.amqp.imageName`.
FMPOV this would be a little easier to understand and document. WDYT?

Reply:
Yes, I would have actually preferred it that way. It is as well what we did in [Ditto](https://github.com/eclipse/ditto/blob/master/deployment/helm/eclipse-ditto/values.yaml). However, it would break the current helm values structure that's why I added a property instead of changing the current one.

However, I can change it if that is ok for you.

Reply:
FMPOV we should do that. We haven't released 1.0 yet so I think we can get away with this change as most people will probably not change the name anyway ...

@dejanb WDYT?

Reply:
Yeah, that sounds good to me

Reply:
@kaizimmerm I think we are in consensus here. So, please change to a single property only ...

Reply:
I have updated the PR. However, I have still a few tests to run. So stay tuned.

Reply:
@sophokles73 my tests are complete. IMHO you could merge.